### PR TITLE
Add StandardJS linting workflow

### PR DESCRIPTION
**Pull request by 0atman**
What
----

Our style of choice is StandardJS. Though there's not much JS here, it should be Standard.

### pre-merge

- [x] Add standard package to npm
- [x] Add `npm lint` and `lint:fix` targets
- [x] Auto-run in a GH action
- [x] Auto-fix the js in this repo, using correct eslint config for our usecase (which for us is `eslint-disable no-var, prefer-const`)

### post merge

- [ ] Check tech docs JS works once built

How to review
-------------

1. The tech docs should still build and look the same.
2. `npm install && npm test` should succeed.

Who can review
--------------

Any devs

### CODE REVIEW

**Comment by 0atman** [comment]
Looks like the standard tests work :ok_hand: 
Screenshot from 2021-05-17 15-14-26

**Review by paroxp — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "standard": "^16.0.3"
+  },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "standard",
+    "lint:fix": "standard --fix"
+  },
+  "standard": {
+    "ignore": [
+      "out/*"
```
Comment (kr8n3r) [inline package.json:12]: `build` -  (`out` is from product pages)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    "env": [
+      "es6",
```
Comment (kr8n3r) [inline package.json:15]: JS here will be all ES5 me thinks

**Review by kr8n3r — APPROVED** [review]
minor changes, otherwise looks good

File: package.json (hunk 2 above)
Comment (0atman) [reply package.json:15]: hmm, no es5 option for standard, I'll remove it.

**Pull request merged**